kpsaliency/datasets/saliencymap.py

```diff
@@ -0,0 +1,161 @@
+# common libraries
+import json
+import os
+import time
+from typing import List, Any, Union, Dict
+from tqdm import tqdm
+
+import numpy as np
+import cv2
+from skimage.morphology import disk
+import multiprocessing as mp
+
+# pvdn dependencies
+from pvdn import PVDNDataset
+
+# own library dependencies
+from kpsaliency.generators import KPBMSGenerator
+from kpsaliency.utils.misc import handle_pbar
+
+
+class SaliencyMapDataset(PVDNDataset):
+    def __init__(self, path: str,
+                 filters: List[Any] = [], transform: List[Any] = None,
+                 read_annots: bool = True, load_images: bool = False,
```

Comment:
load_images should be True as default value. Otherwise the dataset just returns None which cannot be collated by the default collate function

Reply:
done